src/drive/targets/mobile/index.jsx

```diff
@@ -246,6 +196,89 @@ var app = {
       })
     }
   }
-}
 
-app.initialize()
+  startApplication = async () => {
+    console.log('StartApplication', console.trace())
+    if (this.stardedApp) return
```

Comment:
I think we could use `appReady` instead of `stardedApp`, by resolving the promise with `true` for example. Then `if (this.appReady === true)` would work and not require an extra state-tracking variable.